docs/relnotes.md

```diff
@@ -7,18 +7,62 @@ hide:
 
 This is the list of changes to Taipy releases as they were published.
 
-## Community edition: 1.2 (In progress)
+## Community edition: 2.0 (In progress)
+
+Published on TO_BE_DEFINED.
+
+[`taipy` 2.0](https://pypi.org/project/taipy/2.0.0/) contains the latest
+[`taipy-config` 1.0](https://pypi.org/project/taipy-config/1.0.0/),
+[`taipy-gui` 2.0](https://pypi.org/project/taipy-gui/2.0.0/),
+[`taipy-core` 2.0](https://pypi.org/project/taipy-core/2.0.0/) and
+[`taipy-rest` 2.0](https://pypi.org/project/taipy-rest/2.0.0/) packages.
+
+#### Features
+
+**`taipy-core`** 2.0.0
+
+   - TODO SQLDataNodes
+   - TODO SQLTableDatanode
+   - TODO JSONDataNodes
+   - In standalone mode, a job whose status is `SUBMITTED`, `PENDING`, or `BLOCKED`, can be canceled. When
+     canceling the job, its subsequent jobs will be abandoned and their statuses will be set to `ABANDONED`. When
+     the cancel method is called on a job with its status being either `RUNNING`, `COMPLETED`, or `SKIPPED`, its
+     subsequent jobs will be abandoned while its status remains unchanged. A job whose status is `FAILED`,
+     `CANCELED`, or `ABANDONED`, cannot be canceled.
+   - TODO Core as a service
 
 #### Improvements
 
-**`taipy-core`**<br/>1.2.0
+**`taipy-core`** 2.0.0
```

Comment:
Ok so you did not like my carriage return... okok

Reply:
For me the title is the name of the versioned package "core" and "2.0" come together. I don't see the purpose of separating them with a carriage return. But well, that is minor. If you are really convinced and if you insist just one more comment, I will revert without any problem.
:)